Package.swift

```diff
@@ -3,9 +3,19 @@
 
 import PackageDescription
 
+#if os(macOS)
+let exclusions: [String] = [
+    "Classes/CustomLocation",
+    "Classes/EmailSender"
+]
+#else
+let exclusions: [String] = []
+#endif
```

Comment:
Nice 👍

Still, try to define the variable like `excludedSources` to make it clearer (we are not excluding resources here, e.g. images).